prometheus-blackbox-exporter.yaml

```diff
@@ -11,7 +11,6 @@ environment:
     packages:
       - build-base
       - busybox
-      - ca-certificates-bundle
       - curl
       - go
       - wolfi-baselayout
```

Comment:
build-base pulls in wolfi-baselaoyt which pulls in ca-certificates-bundle so you just need build-base